src/main/js/controls/Gantt/helpers/translateHelpers.js

```diff
@@ -121,14 +121,43 @@ const getTrackLabelTransformProperty = (scale, config) => (trackName) =>
  * @private
  * @param {object} config - config object derived from input JSON
  * @param {d3.selection} canvasSVG - d3 selection node of canvas svg
- * @returns {object} d3 svg path
+ * @returns {undefined} - returns nothing
  */
-const translateDefs = (config, canvasSVG) =>
+const translateDefs = (config, canvasSVG) => {
     canvasSVG
         .select(`clipPath#${config.clipPathId}`)
         .selectAll("rect")
         .attr("height", getYAxisHeight(config))
         .attr("width", getXAxisWidth(config));
+    if (
+        config.settingsDictionary.shouldCreateDatelineDefs &&
+        config.dateline.length > 0
+    ) {
+        const shapeHeightArr = [];
+        const shape = d3.selectAll(`.${styles.datelinePoint}`);
+        shape[0].forEach((element) => {
+            const shapeHeight = element.getBBox().height;
+            shapeHeightArr.push(shapeHeight);
+        });
+        const datelineIndicatorHeight = Math.floor(
+            Math.max(...shapeHeightArr) / 2
+        );
+        canvasSVG
+            .select(`clipPath#${config.datelineClipPathId}`)
+            .selectAll("rect")
+            .attr(
+                "height",
+                config.height === 0
+                    ? datelineIndicatorHeight + 5
```

Comment:
This condition is no longer needed is it? We discussed that https://github.com/cerner/carbon-graphs/issues/88 would take care of height not being 0.

Reply:
ok sure I will remove it